homeassistant/components/notify/html5.py

```diff
@@ -182,6 +182,30 @@ def post(self, request):
 
         return self.json_message('Push notification subscriber registered.')
 
+    def delete(self, request):
+        """Delete a registration."""
+        subscription = request.json.get(ATTR_SUBSCRIPTION)
+
+        found = None
+
+        for key, registration in self.registrations.items():
+            if registration.get(ATTR_SUBSCRIPTION) == subscription:
+                found = key
+                break
+
+        if not found:
+            # If not found, unregistering was already done. Return 200
+            return self.json_message('Registration not found.')
+
+        reg = self.registrations.pop(found)
```

Comment:
Trying to delete something that does not exist is still 200, because the server will end up in the desired state (it just already was).